com.adsbynimbus.nimbus/Runtime/Scripts/Nimbus.Internal/Interceptor/ThirdPartyDemand/Vungle/VungleAndroid.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using Newtonsoft.Json;
+using Nimbus.Internal.Utility;
+using OpenRTB.Request;
+using UnityEngine;
+
+[assembly: InternalsVisibleTo("nimbus.test")]
+namespace Nimbus.Internal.Interceptor.ThirdPartyDemand.Vungle {
+	internal class VungleAndroid : IInterceptor, IProvider {
+		private const string NimbusVunglePackage = "com.adsbynimbus.request.VungleDemandProvider";
+		private const string VunglePackage = "com.vungle.ads.VungleAds";
+		private readonly string _appID;
+		private readonly AndroidJavaObject _applicationContext;
+
+		public VungleAndroid(string appID) {
+			_appID = appID;
+		}
+		
+		public VungleAndroid(AndroidJavaObject applicationContext, string appID) {
+			_applicationContext = applicationContext;
+			_appID = appID;
+		}
+		
+		public BidRequest ModifyRequest(BidRequest bidRequest, string data) {
+			if (data.IsNullOrEmpty()) {
+				return bidRequest;
+			}
+			if (bidRequest.User.Ext != null) {
```

Comment:
In the case where ext is null it should be instantiated instead of skipping this step.